axelrod/strategies/shortmem.py

```diff
@@ -0,0 +1,49 @@
+from axelrod import Actions, Player
+from axelrod.actions import Action
+
+C, D = Actions.C, Actions.D
+
+class ShortMem(Player):
+    """
+    A player starts by always cooperating for the first 10 moves.
+    
+    From the tenth round on, the player analyzes the last ten actions, and
+    compare the number of defects and cooperates of the opponent, based in
+    percentage. If cooperation occurs 30% more than defection, it will
+    cooperate.
+    If defection occurs 30% more than cooperation, the program will defect.
+    Otherwise, the program follows the TitForTat algorithm.
+    
+    Names:
+    
+    - ShortMem: [Andre2013]_
+    """
+
+    name = 'ShortMem'
+    classifier = {
+        'memory_depth': 10,
+        'stochastic': False,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    @staticmethod
+    def strategy(opponent: Player) -> Action:
+        
+        if len(opponent.history) <= 10:
+            return C
+        
+        array = opponent.history[:-11:-1]
+        C_counts = array.count('C')
+        D_counts = array.count('D')
+            
+        if C_counts - D_counts >= 3:
```

Comment:
Another question (not sure how we missed this), is the logic correct here?

You're looking at the difference in counts (subtraction) but the description is saying that there should be 30% more cooperations than defections? So I'm not sure the logic is actually correct.

I think this test should be:

```
if C_counts / D_counts >= 1.3:
```

That checks that there are 30% more cooperations.